DEPLOYING.md

```diff
@@ -0,0 +1,51 @@
+# Deploying and Development
+
+The VSCode Extension for PRQL relies on the `prqlc` code from the
+main PRQL repo. There are two Github Actions that wacth for updates
+in this directory to build and release the extension to the
+Microsoft Marketplace.
+
+## Deploying/Updating the Extension
+
+When there is a new version of PRQL (`prqlc` in particular):
+
+- In _package.json_, update:
+
+  - `version` to the proper version (from the main PRQL repo)
+  - `dependencies.prqlc` to the same version
+  - (optional) update other dependencies and devDependencies
+
+- Check _.github/workflows/pull-request.yaml_ and _.github/workflows/release.yml_ to ensure that the `node-version` is correct.
+
+- Run `npm install` to get/update dependencies
+
+- Run `npm run compile`. to build the extensions files
+
+- (I AM GUESSING HERE...) The `.github` workflows check the
+  `main` branch on a daily basis to see whether there have been
+  changes and push a new extension as needed.
```

Comment:
```suggestion
- Run `npm install` to update the `package-lock.json`
- Create a new release. This will start a workflow to release the current version to the VS Code Marketplace
```

Reply:
I don't know how to 'create a new release' - that's part of the reason I want to document it here...